pkgs/top-level/ocaml-packages.nix

```diff
@@ -219,7 +219,12 @@ let
 
     dune = callPackage ../development/tools/ocaml/dune { };
 
-    dune_2 = callPackage ../development/tools/ocaml/dune/2.nix { };
+    dune_2 =
+      if lib.versionAtLeast ocaml.version "4.7"
```

Comment:
Should be "4.07"

Reply:
Fixed. But I think it does not change anything.